source/view/blocklyUI.js

```diff
@@ -31,6 +31,9 @@ function setUpBlocklyPeripherals() {
     var procedureIndicator = document.createElement( "div" );
     var indicatorHighlighter = document.createElement( "div" );
 
+    var toolboxDivs =  document.getElementsByClassName( "blocklyToolboxDiv" );
+    var blocklyToolboxDiv = toolboxDivs[0];
```

Comment:
This is undefined. It seems the blockly toolbox is not being created with the main blockly window.